### PR TITLE
added last updated in accounts array

### PR DESCRIPTION
**Pull request by sid370**
- last updated is added to the accounts array

### CODE REVIEW

File: docs/bank-connect/rest-api.md [hunk 1]
```diff
@@ -168,6 +169,7 @@ The response has the following fields:
     | account_category | string | account category, can be `individual` or `corporate` |
     | account_number | string | account number |
     | credit_limit   | Integer |  limit up to which a company can withdraw from the working capital limit sanctioned |
+    | last_updated | Timestamp |  Timestamp when the account was last updated|
```
Comment (shouryaps) [inline docs/bank-connect/rest-api.md:172]: 1. specify the format of timestamp here
2. specify what last updated means in context of account, as in the account where last statement was uploaded to

File: docs/bank-connect/rest-api.md [hunk 2]
```diff
@@ -135,7 +135,8 @@ On fetching information successfully, the response would be of the following for
             "account_number": "Account Number Extracted",
             "account_category": "individual",
             "credit_limit": 0,
-            "bank": "axis"
+            "bank": "axis",
+            "last_updated": "2023-03-04T07:37:27.976531Z"
```
Comment (shouryaps) [inline docs/bank-connect/rest-api.md:139]: put an example with 24 hour time everywhere for more clarity

**Review by shouryaps — CHANGES_REQUESTED** [review]

File: docs/bank-connect/rest-api.md (hunk 1 above)
Comment (sid370) [reply docs/bank-connect/rest-api.md:172]: resolved

File: docs/bank-connect/rest-api.md (hunk 2 above)
Comment (sid370) [reply docs/bank-connect/rest-api.md:139]: is this needed? the time format is yyyy-MM-ddTHH:mm:ss.SSSZ. It is actually a 24H format

Comment (shouryaps) [reply docs/bank-connect/rest-api.md:139]: if you have mentioned format its okay

File: docs/bank-connect/rest-api.md (hunk 1 above)
Comment (shouryaps) [reply docs/bank-connect/rest-api.md:172]: put the explanation in this table itself instead of in the bullet points

**Review by shouryaps — APPROVED** [review]

**Pull request merged**